replica/memtable.cc

```diff
@@ -693,8 +693,8 @@ partition_snapshot_ptr memtable_entry::snapshot(memtable& mtbl) {
     return _pe.read(mtbl.region(), mtbl.cleaner(), _schema, no_cache_tracker);
 }
 
-flat_mutation_reader_v2
-memtable::make_flat_reader(schema_ptr s,
+std::optional<flat_mutation_reader_v2>
```

Comment:
Please see if optimized_optional is applicable here.

Reply:
It does, we have `flat_mutation_reader_opt` (and v2 equivalent for sure).

Reply:
Done.